packages/variant-theme/src/ThemeGenerator.ts

```diff
@@ -0,0 +1,426 @@
+/* eslint-disable */
+/* tslint-disable */
+//@ts-nocheck
+import {
+  getColorFromString,
+  IColor,
+  updateA
+} from '@fluentui/react';
```

Comment:
we cannot use transitive dependencies 

- update peerDeps for this project to accomodate `@fluentui/react`
- install react to monorepo `yarn add -DW @fluentui/react`

Reply:
fixed